src/Interpreters/loadMetadata.cpp

```diff
@@ -201,6 +204,110 @@ static void loadSystemDatabaseImpl(ContextMutablePtr context, const String & dat
     }
 }
 
+static void convertOrdinaryDatabaseToAtomic(ContextMutablePtr context, const DatabasePtr & database)
```

Comment:
Maybe we can make some command for that? `Alter database engine` or `system` something?

Reply:
Example https://kb.altinity.com/engines/altinity-kb-atomic-database-engine/how-to-convert-ordinary-to-atomic/

Reply:
> Maybe we can make some command for that? Alter database engine or system something?

Yes, I will add it later. The problem is that it's much easier (and safer) to convert database on server startup. Maybe we can create a file in `flags/` directory on "alter database engine" query and actually convert it on next restart.

Reply:
> Example https://kb.altinity.com/engines/altinity-kb-atomic-database-engine/how-to-convert-ordinary-to-atomic/

I have better example =)
https://github.com/ClickHouse/ClickHouse/blob/880780c7a8e4f20afeb06987a48afc7e8dbbf4ae/tests/queries/0_stateless/01155_rename_move_materialized_view.sql